package.json

```diff
@@ -32,19 +32,23 @@
     "url": "https://github.com/CrowdStrike/eslint-config-crowdstrike/issues"
   },
   "homepage": "https://github.com/CrowdStrike/eslint-config-crowdstrike#readme",
+  "engines": {
+    "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
```

Comment:
I'm not sure we need to specify this, since we don't bundle our own eslint version.

Reply:
Needed for `eslint-plugin-n` but I think also needed to run v9 in this repo?